extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -38,5 +67,53 @@ describes.realWin(
       expect(() => shoppingConfig.layoutCallback()).to.not.throw();
       expect(shoppingConfig.isLayoutSupported(Layout.NODISPLAY)).to.be.true;
     });
+
+    it('throws on no config', async () => {
+      expectAsyncConsoleError(async () => {
+        expect(async () => {
+          await shoppingConfig.getConfig();
```

Comment:
We should not call methods of the component directly other than `buildCallback` or `layoutCallback`.
The tests should be independent of implementation.